### ISSUE TITLE
Does not scan DataMatrix format

### ISSUE THREAD

**Issue by kaciula** [juliansteenbakker/mobile_scanner#397]
I am running the example app and it cannot detect DataMatrix codes. I'm attaching some DataMatrix images for testing purposes.

I have also tried using the scan_window branch implementation and example. Using the scan window, the plugin manages to recognize the DataMatrix codes as long as there is nothing else in the scan window apart from the barcode. It works better on Android than on iOS. I believe it has to do with resolution and camera focus. The scan window needs to be clearly focused in order for it to work.

Bottom line, when can we expect the scan window implementation to be released. Scanning DataMatrix only works with the scan window.

[reteta1.pdf](https://github.com/juliansteenbakker/mobile_scanner/files/10165769/reteta1.pdf)
[reteta2.pdf](https://github.com/juliansteenbakker/mobile_scanner/files/10165771/reteta2.pdf)

**Comment by jakusb** [comment]
Same here. I am now forced to use another plugin just to be able to scan datamatrix. I would love to have this included here.

**Comment by kaciula** [comment]
@jakusb What plugin do you use for scanning DataMatrix? Thanks.

**Comment by jakusb** [comment]
I am now looking at flutter_barcode_scanner, it seems to work.
I might consider using that library for all QR/Barcode/DataMatrix.

**Comment by jakusb** [comment]
Downside of flutter_barcode_scanner seems to be that it takes full control of the phone. 
I need to be able to just present a small viewfinder, like I can with MobileScanner...

**Comment by juliansteenbakker** [comment]
I cant do anything because this is package is based on MLKit from google. See https://developers.google.com/ml-kit/vision/barcode-scanning
